src/main/java/org/testng/TestNG.java

```diff
@@ -1243,7 +1243,7 @@ private void generateReports(List<ISuite> suiteRunners) {
    */
   public List<ISuite> runSuitesLocally() {
     if (m_suites.isEmpty()) {
-      setStatus(HAS_NO_TEST);
+      this.m_hasTests = false;
```

Comment:
Don't need it because it is false by default.

Reply:
Agreed. Fixed.